Maquetado/index.html

```diff
@@ -0,0 +1,20 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body>
+  <pre><h1 class="title">Título del libro<h1 class='gen'> (género)</h1></h1> </pre>
+  <br>
+  <br>
+  <pre><h2 class="secondTitle">Autor del libro:<h2 class="bookData" > Nombre del autor del libro</h2></h2></pre>
+  <br>
+<pre><h2 class="secondTitle">Editorial:<h2 class="bookData"> Nombre de la editorial</h2></h2></pre>
+  <br>
+<pre> <h2 class="secondTitle">Año de publicación:<h2 class="bookData"> Año de publicación</h2></h2></pre>
```

Comment:
```secondTitle``` isn't a good name. It could be ```label``` or ```title```.

```bookData``` isn't a good name. The word ```Data``` doesn't say anything.

Remember to work with a formatted file. This file isn't formatted.